src/super_gradients/training/transforms/transforms.py

```diff
@@ -1277,21 +1318,40 @@ def _filter_box_candidates(box1, box2, wh_thr=2, ar_thr=20, area_thr=0.1):
     return (w2 > wh_thr) & (h2 > wh_thr) & (w2 * h2 / (w1 * h1 + 1e-16) > area_thr) & (ar < ar_thr)  # candidates
 
 
-def _mirror(image, boxes, prob=0.5):
+def _flip_horizontal_image(image: np.array):
+    """
+    Horizontally flips image
+    :param image: (np.array) image to be flipped.
+    :return: flipped_image
     """
-    Horizontal flips image and bboxes with probability prob.
+    return image[:, ::-1]
+
 
+def _flip_horizontal_boxes(boxes: np.array, img_width: int):
+    """
+    Horizontally flips bboxes
+    :param boxes: (np.array) bboxes to be flipped. (xyxy format)
+    :return: flipped_boxes
+    """
+    return img_width - boxes[:, [2,1,0,3]]
```

Comment:
Please add a unit test for this method since it looks wrong to me.
You are selecting `X1,Y1,X2,Y2` in the right order via indexing `[2,1,0,3]` into `X2, Y1, X1, Y2`. However you then subtract from img_width all four components. So final boxes are:
`img_width - X2, img_width - Y1, img_width - X1, img_width - Y2`
Y coordinate should remain unchanged during flip operation.